guides/doc-Upgrading_and_Updating/topics/upgrading_external_database.adoc

```diff
@@ -0,0 +1,21 @@
+[id="Upgrading_External_Database_{context}"]
+= Upgrading External Database
+
+You can upgrade an external database from {RHEL} 7 to {RHEL} 8 while upgrading {Project} from {ProjectVersionPrevious} to {ProjectVersion}.
+
+.Prerequisites
+Create a new {RHEL} 8 based host for PostgreSQL server that follows the external database on {RHEL} 8 documentation.
+
+.Procedure
+. Copy the backup database to the new host.
+. Restore the backup database on the new host.
+
+If you keep the same hostname on the new PostgreSQL server, you need to update the DNS preventing the need to do  updates on {Project} itself.
+If the new PostgreSQL server has a new host name or a different CNAME, you need to run the following on the {Project}:
```

Comment:
```suggestion
If the new PostgreSQL server has a new host name or a different CNAME, you need to run the following on {Project}:
```

Maybe reword to "update the reference to the external database on {Project}" or similar? It should IMHO also be a step (aka. start with ". ").